src/components/InsidePostHog/Merch.tsx

```diff
@@ -28,15 +28,15 @@ export default function Merch() {
     return (
         <div>
             <div className="bg-white dark:bg-accent-dark p-4 border border-light dark:border-dark mt-4 mb-0 rounded">
-                <h3 className="text-lg text-center italic leading-tight">
-                    "Some of the best company swag I've ever seen"
+                <h3 className="text-sm text-center italic leading-tight">
+                    "This merch store has some of the best company swag I've ever seen"
                 </h3>
                 {/* quote source: https://posthog.slack.com/archives/C011L071P8U/p1710758940243199 */}
 
                 <Link to="/merch">
                     <CloudinaryImage
-                        src="https://res.cloudinary.com/dmukukwp6/image/upload/v1715109076/dw.jpg"
-                        alt="PostHog t-shirt"
+                        src="https://cdn.shopify.com/s/files/1/0452/0935/4401/files/DSC07383_3265x4897_crop_center.jpg?v=1709570954"
```

Comment:
Are y'all positive we can use a a `shopify` image as a prop to the `CloudinaryImage` component? I'd expect it to require a cloudinary link - which could be required at the Typescript level, btw

Reply:
Good call I hadn't noticed that. It looks like it works as far as I can tell here:
<img width="1437" alt="community-page" src="https://github.com/user-attachments/assets/d1aba4be-0251-4242-ab75-a2563a251340" />


Any thoughts if I should switch to an `img` element or leave it as is?

Reply:
@bijanbwb The Cloudinary component returns a standard image element if the URL passed isn't a Cloudinary URL, so you're technically good! That image is massive, though 👀 [PR here](https://github.com/PostHog/posthog.com/pull/10336)

@rafaeelaudibert Added some basic types here to encourage Cloudinary URLs [here](https://github.com/PostHog/posthog.com/pull/10337)!

Reply:
Thanks guys :)